snarkos/ledger/server.rs

```diff
@@ -262,6 +299,28 @@ impl<N: Network> Server<N> {
         Ok(reply::with_status(reply::json(&records), StatusCode::OK))
     }
 
+    /// Returns the number of peers connected to the node.
+    async fn peers_count(ledger: Arc<Ledger<N>>) -> Result<impl Reply, Rejection> {
+        Ok(reply::json(&ledger.peers.read().len()))
+    }
+
+    /// Returns the peers connected to the node.
+    async fn peers_all(ledger: Arc<Ledger<N>>) -> Result<impl Reply, Rejection> {
+        Ok(reply::json(
+            &ledger.peers.read().keys().map(|addr| addr.ip()).collect::<Vec<std::net::IpAddr>>(),
```

Comment:
it might be preferable to return the full addresses (including ports), as other peers might not use the default one

Reply:
+1